doc/rm/comportability_specification/index.md

```diff
@@ -299,6 +299,7 @@ The following standardised countermeasures are defined:
 | BKGN_CHK       | The asset is protected with a continuous background health check |
 | GLITCH_DETECT  | The asset is protected by an analog glitch detector | CTRL, FSM, CLK, RST
 | SW_UNREADABLE  | The asset is not accessible by software |
+| SIDELOAD       | The asset is can be loaded without exposing it to software | KEY
```

Comment:
@vogelpi 
FYI: I took this from your AES CM label PR

Reply:
I think you want to drop the "is" here?

Reply:
Sounds good, I will rebase the AES PR later today.